### DISCUSSION TITLE
Where do i change extension gallery url?

### DISCUSSION THREAD

**Discussion by DWShuo** [coder/code-server#4752] [Q&A] [answered]
Where do i change the url for the extension gallery?
The FAQ says to set `$EXTENSIONS_GALLERY`, am i setting this in the yaml config file?

I see it also says `export EXTENSIONS_GALLERY='{"serviceUrl": "https://extensions.coder.com/api"}'`, is there a way to set the url in the config file instead of exporting it to your environment?

**Comment by jsjoeio** [comment, marked as answer]
AFAIK, the only way to change it is to set it as an environment variable.

**Comment by Almotaek** [comment]
Hey, which environment file?
